global.json

```
@@ -1,13 +1,14 @@
 {
   "tools": {
-    "dotnet": "3.1.200",
+    "dotnet": "5.0.100-preview.8.20417.9",
```

The build is failing due to missing a 3.1 runtime. I tested that 

```suggestion
    "dotnet": "5.0.100-preview.8.20417.9",
    "runtimes": {
      "dotnet": [
        "3.1.7"
      ]
    },
```
fixed this locally for me.